src/test/java/Homework23/LoginPage23.java

```diff
@@ -0,0 +1,49 @@
+package Homework23;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginPage23 extends BasePage23 {
+
+    @FindBy(css="[type='submit']")
+            WebElement submitButtonLocator;
+
+    @FindBy(css = "[type='email']")
+            WebElement emailField;
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)